code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2092,3 +2092,20 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "planet_cracker"
 	glass_name = "Planet Cracker"
 	glass_desc = "Although historians believe the drink was originally created to commemorate the end of an important conflict in man's past, its origins have largely been forgotten and it is today seen more as a general symbol of human supremacy."
+	
+/datum/reagent/consumable/ethanol/cactuscooler
+	name = "Cactus Cooler"
+	description = "an alcoholic drink created by fermenting cactus, its color is odd looking"
```

Comment:
"**A**n alcoholic drink created by fermenting cactus, its color is odd looking**.**"